modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -185,11 +185,14 @@ endif::[]
 
 | `provisioningNetwork`
 |
-| Set this parameter to `Disabled` to disable the requirement for a `provisioning` network. User may only do virtual media based provisioning, or bring up the cluster using assisted installation. If using power management, BMC's must be accessible from the machine networks. User must provide two IP addresses on the external network that are used for the provisioning services.
+| The `provisioningNetwork` configuration setting determines whether the cluster uses the `provisioning` network. If it does, the configuration setting also determines if the cluster manages the network.
+
+* `Disabled`: Set this parameter to `Disabled` to disable the requirement for a `provisioning` network. When set to `Disabled`, you may only use virtual media based provisioning, or bring up the cluster using the assisted installer. If `Disabled` and using power management, BMCs must be accessible from the `baremetal` network. If `Disabled`, you must provide two IP addresses on the `baremetal` network that are used for the provisioning services.
```

Comment:
What is the purpose of the asterisk at the beginning of this sentence? Was it meant to be a bullet? If so, it's not rendering as bullets - you'll need to add an "a" to the columns for the table. Something like:

```
[cols="1,1,2a", options="header"]
````

Reply:
Removed asterisks. I think it looks better that way.